backend/src-common/src/main/java/org/eclipse/sw360/datahandler/db/RepositoryUtils.java

```diff
@@ -59,4 +61,8 @@ public static RequestSummary doBulk(Collection<?> objects, User user, DatabaseRe
 
         return searchIds;
     }
+
+    public static boolean isDuplicateAttachment(Set<Attachment> attachments) {
```

Comment:
Maybe place this method into the AttachmentConnector, which seems to be used for attachment related methods? As this method has no functionality that can be used for different data types, I am not sure if this universal class is a good place.

Reply:
Done.Thanks for mentioning that.